.gitignore

```diff
@@ -21,3 +22,9 @@ env
 */_build
 */_markdown
 */visualizations
+
+data/example_dataset/metaclustering/*output*.csv
+data/example_dataset/metaclustering/example_manually_adjust_metaclusters.ipynb
```

Comment:
This notebook should get moved to /templates and added to git

Reply:
Notebook is already in templates, this was just a quick solution for the opposite of `update_notebooks.sh` from a example notebook developer perspective.

I would love to discuss a better way to to this, honesty my method feels silly, but it worked and was simple.

Use case:
As a developer, I want changes to ipynb files to be immediately reflected in `git status` so that I choose to commit or discard those changes as I commit changes to supporting library files.

My simple solution is shown below, however, it requiree the additional gitignores to prevent tracking the intermediate file.

![image](https://user-images.githubusercontent.com/6101677/134819603-c3c28e8d-fcf8-4163-8f20-b5a3b8411b14.png)

/speculation
Is the reason for copying the templates during docker build something like this:
As data analyst, I don't want any of my changes to ipynb files to be tracked by git so that pulling updates to the notebooks doesn't cause merge conflicts and I can manually pull latest upstream ipynb files using `update_notebooks.sh`

Reply:
Just thought of a more robust solution...and it's even simpler.

Add an optional arg to "start_docker.sh" e.g. `--develop-template-notebooks`

Which will bind the templates folder directly. I think this would allow for having the "favor upstream" workflow be the default behavior while also allowing for normal edit-add-commit workflow when developing template notebooks.

Reply:
Yes, the idea is that the vast majority of people using the repo are not developers, and we don't want them to run into merge conflicts when they pull new changes. Mounting the templates directly would solve the problem, or working on a copy that's somewhere else until the PR is ready to merge in. @vacuousplanet/ @alex-l-kong  any preference?